build/pack-conda.ps1

```diff
@@ -19,7 +19,7 @@ $PSVersionTable | Format-Table | Out-String | Write-Host;
 function Pack-CondaRecipe() {
     param(
         [string] $Path,
-        [string[]] $PythonVersions = @("3.7")
+        [string[]] $PythonVersions = @("3.7", "3.8")
```

Comment:
Let's see if we can include 3.9 as it's the default for miniconda.
```suggestion
        [string[]] $PythonVersions = @("3.7", "3.8", "3.9")
```

Reply:
Agreed, I realize I forgot to push my commit with this change. Thanks for the reminder!